.github/workflows/cmake.yml

```diff
@@ -0,0 +1,90 @@
+# inspired by "build GDAL library and to run test with CMake"
+# from https://github.com/OSGeo/gdal/blob/master/.github/workflows/cmake_builds.yml
+
+---
+name: CMake
+
+on:
+    push:
+        paths-ignore:
+            - 'doc/**'
+        branches:
+            - main
+            - releasebranch_*
+    pull_request:
+        paths-ignore:
+            - 'doc/**'
+
+concurrency:
+    group: ${{ github.workflow }}-${{ github.head_ref || github.ref }}
+    cancel-in-progress: true
+
+env:
+    CMAKE_UNITY_BUILD: OFF
+
+permissions:
+    contents: read  #  to fetch code (actions/checkout)
+
+jobs:
+
+    build-cmake:
+        runs-on: ubuntu-22.04
+        env:
+            PythonVersion: 3.9
+        steps:
+            - name: Checkout GRASS
+              uses: actions/checkout@v4.1.1
+            - name: Set up Python
+              uses: actions/setup-python@42375524e23c412d93fb67b49958b491fce71c38 # v5.4.0
+              with:
+                  python-version: ${{ env.PythonVersion }}
+            - name: Install CMake 3.16.0
+              run: |
+                cd ${GITHUB_WORKSPACE}
+                wget https://github.com/Kitware/CMake/releases/download/v3.16.0/cmake-3.16.0-Linux-x86_64.tar.gz
+                tar xzf cmake-3.16.0-Linux-x86_64.tar.gz
+                echo "CMAKE_DIR=$GITHUB_WORKSPACE/cmake-3.16.0-Linux-x86_64/bin" >> $GITHUB_ENV
+                echo "$GITHUB_WORKSPACE/cmake-3.16.0-Linux-x86_64/bin" >> $GITHUB_PATH
```

Comment:
"3.16.0" is at 3 different places, make it a variable.

Reply:
good point, updated.